src/components/InvitationModal/stylesheet.scss

```diff
@@ -0,0 +1,245 @@
+.invitation-modal {
+    margin: -20px;
+}
+
+.invitation-modal-content {
+    text-align: center;
+    color: white;
+}
+
+.email-block {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-evenly;
+    padding-bottom: 5px;
+}
+
+// form {
+//     display: flex;
+//     flex-direction: column;
+// }
+
+//Style for email input
+
+.email-input {
+    display: flex;
+    flex-direction: column;
+}
+
+input {
+    width: 282px;
+    height: 24px;
+}
+
+input[type=email] {
+    padding: 15px;
+    border-radius: 6px;
+    background-color: rgb(35, 35, 35);
+}
+
+input[type=email]:focus {
+    border: 1px solid rgba(255, 255, 255, 0.5);
+    outline: none
+}
```

Comment:
All of these could be combined. There's just 1 input anyway. So something like
```
input {
    // styles

    &:focus {
         // more styles
    }
}
```

Check for similar occurrences to make your code more compact.